queue/external_typedefs.h

```diff
@@ -2,11 +2,6 @@
 #define __QUEUE_EXTERNAL_TYPEDEFS_H__
 
 
-/*
- * An enqueueable item with a generic data payload.
- */
-typedef void queue_item_t;
-
 /*
  * A queue of items with a generic data payload.
```

Comment:
The doc comment of the `queue_t` type needs updating after making the `queue_item_t` internal, as it still mentions the queue using queue items for storage.

Reply:
Done.